From 730f0b2ae41d08476fec71d6c3b06fb856b80d01 Mon Sep 17 00:00:00 2001
From: Tim Foster <tim.foster@joyent.com>
Date: Wed, 26 Jun 2019 09:03:33 +0100
Subject: [PATCH] OS-7822 build_jenkins shouldn't run smartos-release for debug
 builds Reviewed by: Trent Mick <trentm@gmail.com> Approved by: Trent Mick
 <trentm@gmail.com>

---
 tools/build_jenkins | 14 +++++++-------
 1 file changed, 7 insertions(+), 7 deletions(-)

diff --git a/tools/build_jenkins b/tools/build_jenkins
index 0660618e..a8ffcefc 100755
--- a/tools/build_jenkins
+++ b/tools/build_jenkins
@@ -173,12 +173,12 @@ case "${PLATFORM_BUILD_FLAVOR}" in
         ;;
 esac
 
-# Update Manta snaplinks for smartos, only if this is a release build
-case "${PLATFORM_BUILD_FLAVOR}" in
-    'smartos'|'triton-and-smartos')
-        if [[ -n "${ENGBLD_RELEASE}" ]]; then
+# Update Manta snaplinks for smartos, only if this is a non-debug release build
+if [[ -z "${ENGBLD_CONFIGURE_DEBUG_ARG}" && -n "${ENGBLD_RELEASE}" ]]; then
+    case "${PLATFORM_BUILD_FLAVOR}" in
+        'smartos'|'triton-and-smartos')
             log_cmd env TRACE=1 ./tools/smartos-release \
                 "${BRANCH}" "${TIMESTAMP}"
-        fi
-        ;;
-esac
+            ;;
+    esac
+fi
-- 
2.21.0

